src/client/pages/Offer/Checkout/index.tsx

```diff
@@ -359,16 +368,17 @@ export const Checkout = ({
   const startSign = async () => {
     setSignUiState('STARTED')
     const { values, initialValues, submitForm } = formik
-    const { firstName, lastName, email, ssn } = values
+    const { firstName, lastName, email, ssn, phoneNumber } = values
+    const isFormDataUpdated =
```

Comment:
Maybe we could use formik `dirty` flag here instead of check if something changed in the form when compared with initial values.

```js
const { values, initialValues, submitForm, dirty } = formik
```
[Reference](https://formik.org/docs/api/formik#dirty-boolean)

Reply:
Amazing! That's much cleaner! I will test it :)